integr8ly/integreatly-operator
# PROW E2E test flake C01_-_Verify_Alerts_are_not_firing_apart_from_DeadMansSwitch

# Description
This helps prevent the following [Jira](https://issues.redhat.com/browse/INTLY-7481)
# Verification
1. run integreatly off this branch
2. Make a few alerts fire
3. Comment out unnesscary tests to save time in the tests.go 
4. Run make/test/functional
5. The test TestIntegreatlyAlertsFiring should fail giving info about the alerts that fired and logs about pods. Including Pod Name, Namespace and status
```
time="2020-05-19T10:45:26+01:00" level=info msg="Podname : rhsso-postgres-rhmi-789fb876c9-v72nr"
time="2020-05-19T10:45:26+01:00" level=info msg="Namespace : redhat-rhmi-operator"
time="2020-05-19T10:45:26+01:00" level=info msg="Status : {Running [{Initialized True 0001-01-01 00:00:00 +0000 UTC 2020-05-18 17:33:09 +0100 IST  } {Ready True 0001-01-01 00:00:00 +0000 UTC 2020-05-18 17:39:51 +0100 IST  } {ContainersReady True 0001-01-01 00:00:00 +0000 UTC 2020-05-18 17:39:51 +0100 IST  } {PodScheduled True 0001-01-01 00:00:00 +0000 UTC 2020-05-18 17:33:09 +0100 IST  }]    10.0.128.57 10.131.0.75 [{10.131.0.75}] 2020-05-18 17:33:09 +0100 IST [] [{rhsso-postgres-rhmi {nil &ContainerStateRunning{StartedAt:2020-05-18 17:39:25 +0100 IST,} nil} {nil nil nil} true 0 registry.redhat.io/rhscl/postgresql-10-rhel7:latest registry.redhat.io/rhscl/postgresql-10-rhel7@sha256:3711a4a27cac087be0934aeb39afe395ac314b76d26198b31a5a5a2aa4e39beb cri-o://3570fa2ee04ce7fd720f73987f8b0f2e851df17df188090044c390d6176cdb8d 0xc00052b76f}] Burstable []}"
```
